node/node.go

```diff
@@ -82,6 +82,8 @@ func MakeNode(config *params.NodeConfig, db *leveldb.DB) (*node.Node, error) {
 			return nil, fmt.Errorf("%v: %v", ErrLightEthRegistrationFailureUpstreamEnabled, err)
 		}
 
+		logger.Info("LES protocol is disabled")
```

Comment:
is it accidental?

Reply:
@dshulyak no, I just realized that in that branch, there was no indication in the log that LES is not enabled, so I figured it would be useful to always have that indication in the log.

Reply:
i think when les is enabled it is written to a log anyway

Reply:
yeah, this is just to make it more obvious when looking at a log file at a glance.